Source/WebCore/dom/MouseEvent.idl

```diff
@@ -44,13 +44,12 @@
     readonly attribute double movementY;
     [ImplementedAs=force] readonly attribute double webkitForce;
 
-    // FIXME: relatedTarget should be of type EventTarget? but we need to use any to support a quirk for iAd Producer (rdar://problem/30640101).
-    [CallWith=CurrentGlobalObject&CurrentScriptExecutionContext, ImplementedAs=initMouseEventQuirk] undefined initMouseEvent([AtomString] DOMString type,
-        optional boolean canBubble = false, optional boolean cancelable = false,
+    // FIXME: Using "undefined" as default parameter value is wrong.
+    undefined initMouseEvent(optional [AtomString] DOMString type, optional boolean canBubble = false, optional boolean cancelable = false,
```

Comment:
This is still wrong. `type` shouldn't be optional. That's also why you regressed WPT tests. (The FIXME is also wrong.)

Reply:
🤦 Don't code quickly when you are sick with headaches. 
Thanks @annevk